inc/consent/namespace.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Altis\Core\Consent;
+
+use Altis;
+use Altis\Consent\Settings;
+
+/**
+ * Kick it off.
+ */
+function bootstrap() {
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\set_consent_defaults' );
+	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_plugins', 1 );
+}
+
+/**
+ * Save the defaults to the database if nothing has been set yet.
+ */
+function set_consent_defaults() {
+	$config  = Altis\get_config()['modules']['core']['consent'];
+	$options = get_option( 'cookie_consent_options' );
+
+	// Bail if we've turned consent off explicitly.
+	if ( $config === false ) {
+		return;
+	}
+
+	// Bail if options have been set.
+	if ( $options ) {
+		return;
+	}
```

Comment:
I think this might be confusing, in my mind even if I'd set some options if I was to then configure it I'd expect the configuration to take precedence. It'd be hard to debug what was happening and why otherwise.

Reply:
Yep, the way I wrote this originally was setting things in the config was a default value, but it's been refactored now.